plugin-server/src/cdp/services/hog-executor.service.ts

```diff
@@ -94,15 +94,15 @@ export const formatHogInput = (bytecode: any, globals: HogFunctionInvocationGlob
                     ...res,
                 }
             }
-        } else {
-            ret = {
-                ...Object.fromEntries(
-                    Object.entries(bytecode).map(([key2, value]) => [
-                        key2,
-                        formatHogInput(value, globals, key ? `${key}.${key2}` : key2),
-                    ])
-                ),
-            }
+        }
+
+        ret = {
```

Comment:
I think this is getting really hard to read... Now that we have a return object I think we should just do 

```
		for (const [subkey, value] of Object.entries(bytecode)) {
            if (subkey === EXTEND_OBJECT_KEY) {
                continue
            }
            ret[subkey] = formatHogInput(value, globals, key ? `${key}.${subkey}` : subkey)
        }
```

Also reduces it from like 4 loops to 1